app/Http/Controllers/api/StudentBootcampDetailController.php

```diff
@@ -1,50 +1,35 @@
 <?php
 
+declare(strict_types=1);
+
 namespace App\Http\Controllers\api;
 
+use Exception;
 use App\Http\Controllers\Controller;
-use App\Models\Student;
+use App\Service\Student\StudentBootcampDetailService;
+use App\Exceptions\StudentNotFoundException;
+use App\Exceptions\ResumeNotFoundException;
+use App\Exceptions\ResumeBootcampNotFoundException;
+use Illuminate\Http\JsonResponse;
 
 class StudentBootcampDetailController extends Controller
 {
-    public function __invoke($uuid)
-    {
-        // Student::where('id', $uuid)->firstOrFail();
-
-        $bootcamp_detail = [
-
-            'bootcamp' => [
-                [
-                    'bootcamp_id' => '1',
-                    'bootcamp_name' => 'Full Stack PHP',
-                    'bootcamp_end_date' => ['November 2023'],
-                ],
-                [
-                    'bootcamp_id' => '2',
-                    'bootcamp_name' => 'Back End Nodejs',
-                    'bootcamp_end_date' => ['December 2023'],
-                ],
-                [
-                    'bootcamp_id' => '3',
-                    'bootcamp_name' => 'Data Analytics',
-                    'bootcamp_end_date' => ['January 2024'],
-                ],
-                [
-                    'bootcamp_id' => '4',
-                    'bootcamp_name' => 'Front End Angular',
-                    'bootcamp_end_date' => ['December 2022'],
-                ],
-                [
-                    'bootcamp_id' => '5',
-                    'bootcamp_name' => 'Front End React',
-                    'bootcamp_end_date' => ['October 2022'],
-                ],
-            ]
-        ];
-        
-        $random_bootcamp_index = array_rand($bootcamp_detail['bootcamp']);
-        $random_bootcamp = $bootcamp_detail['bootcamp'][$random_bootcamp_index];
+    private StudentBootcampDetailService $studentBootcampDetailService;
 
-        return response()->json($random_bootcamp);
+    public function __construct(StudentBootcampDetailService $studentBootcampDetailService)
+    {
+        $this->studentBootcampDetailService = $studentBootcampDetailService;
+    }
+    public function __invoke($studentId): JsonResponse
+    {
+        try {
+            $service = $this->studentBootcampDetailService->execute($studentId);
+            return response()->json($service);
+        } catch (StudentNotFoundException | ResumeNotFoundException | ResumeBootcampNotFoundException $e) {
```

Comment:
In line with what we've been discussing among colleagues these days, if you decide that the absence of bootcamps returns an empty array, perhaps the ResumeBootcampNotFoundException exception may not be necessary, as it seems that your service will not throw it.